README.md

```diff
@@ -61,8 +63,34 @@ below.
 
     The default process will install ``libAcesContainer.so`` to ``/usr/local/lib``
     and a number of header files into ``/usr/local/include/aces``
-
-
+    
+    
+		
+               
+* Windows 
+
+	* x86	
+	    
+	    From the root source directory:
+
+			$ mkdir build && cd build
```

Comment:
Thanks for the suggestion! Just update README again.



[Δ]<http://www.oscars.org>


Miaoqi Zhu
Lab Systems Administrator


Academy of Motion Picture Arts and Sciences
Science & Technology Council
1313 Vine Street • Hollywood, CA 90028


310.247.3000 x3302 • MZhu@oscars.org<mailto:MZhu@oscars.org>




From: Alexander Forsythe <notifications@github.com<mailto:notifications@github.com>>
Reply-To: ampas/aces_container <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Wednesday, June 22, 2016 at 2:50 PM
To: ampas/aces_container <aces_container@noreply.github.com<mailto:aces_container@noreply.github.com>>
Cc: Miaoqi Zhu <mzhu@oscars.org<mailto:mzhu@oscars.org>>, Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: Re: [ampas/aces_container] Window build support (#6)


In README.md<https://github.com/ampas/aces_container/pull/6#discussion_r68142005>:

> @@ -61,8 +63,34 @@ below.
>
>      The default process will install ``libAcesContainer.so`` to ``/usr/local/lib``
>      and a number of header files into ``/usr/local/include/aces``
> -
> -
> +
> +
> +
> +
> +* Windows
> +
> +     * x86
> +
> +         From the root source directory:
> +
> +                     $ mkdir build && cd build


The windows build instructions should include windows commands rather than *nix commands.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ampas/aces_container/pull/6/files/db438ac8e809d51f2ee160e5b090e3505a423eb0#r68142005>, or mute the thread<https://github.com/notifications/unsubscribe/AGmu71ZcywKz4IQlQALTf9b7ProHEHoRks5qOa4kgaJpZM4I8JWw>.